installation/install.sh

```
@@ -1961,7 +1960,7 @@ iz_check_space(){
 
     # Check the /tmp directory have enough space to store unpackaged zstack.war and pip tmp files
     zstack_war_size_B=`unzip -l $upgrade_folder/zstack.war | tail -n1 | awk '{ print $1 }'`
-    (( zstack_war_size_KiB = $zstack_war_size_B / 1024 ))
+    (( zstack_war_size_KiB = $zstack_war_size_B // 1024 ))
```

Shellcheck (0.10.0)

[error] Shells disambiguate (( differently or not at all. For subshell, add spaces around ( . For ((, fix parsing errors.

(SC1105)